package.json

```diff
@@ -32,6 +33,7 @@
   },
   "devDependencies": {
     "ava": "^0.14.0",
+    "chokidar": "^1.6.1",
```

Comment:
Yeah, `ava` asked for it to have `watch` support.
I could remove both `chokidar`and the `ava:watch` npm script